tracer/test/Datadog.Trace.Tests/Configuration/TracerSettingsTests.cs

```diff
@@ -1234,6 +1234,48 @@ public void InitializeHeaderTag(string tagName, bool headerTagsNormalizationFixE
             finalTagName.Should().Be(expectedTagName);
         }
 
+        [Theory]
+        [InlineData(null, null, "555")]
+        [InlineData(null, "400", "400")]
+        [InlineData("444", null, "444")]
+        [InlineData("444", "424", "444")]
+        public void ValidateServerErrorStatusCodes(string newServerErrorKeyValue, string deprecatedServerErrorKeyValue, string expectedServerError)
```

Comment:
nipticky:

To me this test data reads strangely - `null`, `null`, `"555"`
Where the reality is the "expected` server errors is actually the default range - not just a single value.
Could we maybe assert on that range?

Reply:
Same as previous comment since the only thing changed was the name I hoped adding the test for asserting the precedence was enough since they check for the ranges in other tests too: https://github.com/DataDog/dd-trace-dotnet/blob/7fa9adde56eb12badda28c82929e7f14d56b8d82/tracer/test/Datadog.Trace.Tests/Configuration/TracerSettingsTests.cs#L178-L193

But I can add a quick check for the range here as I tried that before but changing it to keep it simpler.

Reply:
Oh I see :)

Now I'm wondering if it'd make more sense to re-use that same data.
But I'd be fine with whatever 👍

Reply:
So since testing the range made sense added it here: https://github.com/DataDog/dd-trace-dotnet/pull/6095/commits/416297a5b77ded91243d5643258ea159bf81b461